modules/virt-enabling-preallocation-for-dv.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-using-preallocation-for-datavolumes.adoc
+
+[id="virt-enabling-preallocation-for-dv_{context}"]
+= Enabling preallocation for a data volume
+
+If preallocation is not globally enabled for the cluster, you can enable it for specific data volumes by including the `spec.preallocation` field in the data volume manifest. This can be done in either the web console or when using the OpenShift client (oc).
+
+Preallocation mode is supported for all CDI source types.
+
+.Procedure
+
+* Specify the `spec.preallocation` field in the data volume manifest:
```

Comment:
maybe s/Specify/Edit

Reply:
or ". Edit the `spec.config.preallocation` field so that it has a value of `true`:" as you wrote in the other module :)

Reply:
It could be a new DV so I'm going to leave it as 'Specify', which seems to have wide usage throughout